README.md

```diff
@@ -2,24 +2,58 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+# Directory structure
+
+```
+task-manager-app/
+   README.md
+   node_modules/
```

Comment:
node_modules are not necessary in project structure doc. Only include files related to development: `src/`,